2015/intro_concurrency.slide

```diff
@@ -0,0 +1,375 @@
+Introduction to Go and Concurrency
+4 June 2015
+
+Brendan Tracey
+
+* Talk Outline
+- Why Go
+- Introduction to programming in Go
+- Concurrent examples
+
+* The Sell
+
+* Scientific Computing Circa 2008
+
+.image intro_concurrency/programming_2008.jpg _ 600
+
+Choice between easy for humans or fast for computers
+
+* Scientific Computing 2015
+
+.image intro_concurrency/programming_2015.jpg _ 600
+
+Go is the knee of the curve
+
+* What is Go?
+
+* What is Go?
+	"Go is an attempt to combine the ease of programming of an 
+	interpreted, dynamically typed language with the efficiency and 
+	safety of a statically typed, compiled language. It also aims to be 
+	modern, with support for networked and multicore computing." http://golang.org/doc/faq
+
+- Compiled
+- Open-Source
+- Statically-Typed
+- Garbage collected
+- Concurrent
+- My language of choice
+
+* Origins
+Designed at Google by 
+
+- Ken Thompson (C, Unix)
+- Rob Pike (UTF-8) 
```

Comment:
Taking out the accolades may just be easier. I was giving examples my audience would be familiar with (not attempting to diminish).